share/cmake/modules/FindExtPackages.cmake

```diff
@@ -105,20 +89,37 @@ if(OCIO_BUILD_PYTHON OR OCIO_BUILD_DOCS)
     endif()
 endif()
 
-# The presence of OpenImageIO allows additional OCIO apps and the OSL
-# translation unit tests to be built.
+# The presence of OpenImageIO or OpenEXR allows additional OCIO apps to be built.
 
 if(OCIO_BUILD_APPS OR OCIO_BUILD_TESTS)
+    # Try to use OpenImageIO first as it supports more formats.
+
     # OpenImageIO
     # https://github.com/OpenImageIO/oiio
-
     set(OIIO_VERSION "2.1.9")
 
     if(OCIO_USE_OIIO_CMAKE_CONFIG)
         find_package(OpenImageIO ${OIIO_VERSION} CONFIG)
     else()
         find_package(OpenImageIO ${OIIO_VERSION})
     endif()
+
+    if(OpenImageIO_FOUND AND TARGET OpenImageIO::OpenImageIO)
+        add_library(OpenColorIO::ImageIOBackend ALIAS OpenImageIO::OpenImageIO)
+        set(OCIO_IMAGE_BACKEND OpenImageIO)
+    else()
+        # OpenEXR
+        # https://github.com/AcademySoftwareFoundation/openexr
+        find_package(OpenEXR 3.1.5)
```

Comment:
Same comment here -- any 3.1.x should do (maybe even any 3.x?), I don't think there's a good reason to insist on 3.1.5 and no earlier.

Reply:
Is there a reason we would go below 3.1 as the VFX reference platform require this version for 2022 which is our target for the next release?

Reply:
In this case, probably not. My impression is that anybody not on 3.1 is probably still stuck on 2.4. I don't think 3.0 was ever important to anybody, since it fell between vfx platform years.

On the other hand, there is almost certainly no difference between 3.0 and 3.1 that is relevant to OCIO, so insisting on 3.1+ rather than 3.0+ has no upside at all for OCIO, while potentially constraining somebody downstream (though probably not, in practice).

Reply:
Thanks for the feedback @lgritz, unless asked otherwise, I'll downgrade to 3.0 is the next round of commits on the PR.